soapui/src/main/java/com/eviware/soapui/impl/wsdl/submit/transports/http/HttpClientRequestTransport.java

```diff
@@ -342,8 +359,9 @@ private ExtendedGetMethod followRedirects(HttpClient httpClient, int redirectCou
             }
 
             try {
-                getMethod = followRedirects(httpClient, redirectCount + 1, getMethod, response, httpContext);
-            } finally {
+				getMethod = followRedirects(httpClient, redirectCount + 1, getMethod, response, httpContext, submitContext);

```

Comment:
It looks like the code formatting is broken here, or is it just GitHub that's misrepresenting your code?